x-pack/plugins/security_solution/public/common/components/event_details/overview/status_popover_button.tsx

```diff
@@ -69,7 +69,11 @@ export const StatusPopoverButton = React.memo<StatusPopoverButtonProps>(
           fieldName={enrichedFieldInfo.data.field}
           linkValue={enrichedFieldInfo.linkValue}
           fieldType={enrichedFieldInfo.data.type}
-          fieldFormat={enrichedFieldInfo.data.format}
+          fieldFormat={
+            typeof enrichedFieldInfo.data.format === 'string'
```

Comment:
Yea, I would just wrap all of these in a helper to be removed at a later time

Reply:
➕ ➕